client/gutenberg/extensions/visited/components/edit.jsx

```diff
@@ -0,0 +1,74 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { memoize } from 'lodash';
+import { Notice, PanelBody, RadioControl, TextControl } from '@wordpress/components';
+import { Component, Fragment } from '@wordpress/element';
+import { InnerBlocks, InspectorControls } from '@wordpress/editor';
+
+/**
+ * Internal dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { CRITERIA_AFTER, CRITERIA_BEFORE } from '../constants';
+
+const getRadioOptions = memoize( threshold => [
+	{ label: sprintf( 'Show after %d views', threshold ), value: CRITERIA_AFTER },
+	{ label: sprintf( 'Show before %d views', threshold ), value: CRITERIA_BEFORE },
+] );
+
+export default class VisitedEdit extends Component {
+	onChangeCriteria = criteria => this.props.setAttributes( { criteria } );
+	onChangeThreshold = threshold => {
+		threshold.length &&
+			Number.isFinite( +threshold ) &&
+			+threshold > 0 &&
+			this.props.setAttributes( { threshold } );
+	};
```

Comment:
Nitpick, we've been trying to make our action handler method names be more descriptive of what they do (rather than what action/event they're connected to) -- so in this case, maybe use `setCriteria` and `setThreshold`?

Reply:
Sounds reasonable to me 👍